### ISSUE TITLE
Feedback: "Navigate forwards to a slider" (Media Seek Slider, Test 1, Command "'f' (virtual cursor active)", V25.03.14)

### ISSUE THREAD

**Issue by jha11y** [w3c/aria-at#1209]
I am getting inconsistent results using f (virtual cursor active). Most times Jaws does not announce min and max values, but occasionally JAWS will announce mix and max values when testing the same exact way.

My JAWS version is actually 2025.2503.39

## Test Setup

- Test File: [test-05-navForwardsToSlider-jaws.collected.html](https://aria-at.netlify.app/tests/seek-slider/test-05-navForwardsToSlider-jaws.collected.html)
- AT: JAWS (Version 2025.2412.50)
- Browser: Chrome (Version 134)
- Command: `'f' (virtual cursor active)`

**Comment by IsaDC** [comment]
Hi @jha11y 
As those values should not be announced, I'm closing this issue in favor of #1215, where I've consolidated both our results for further review.
